src/app/app.module.ts

```diff
@@ -9,20 +9,28 @@ import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
 import { HomeModule } from './home/home.module';
 import { LayoutModule } from './layout/layout.module';
+import { AboutUsModule } from './about-us/about-us.module';
+import { EventsComponent } from './events/events.component';
+import { BlogComponent } from './blog/blog.component';
+import { GetInvolvedComponent } from './get-involved/get-involved.component';
 
 export function HttpLoaderFactory(http: HttpClient) {
   return new TranslateHttpLoader(http);
 }
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    EventsComponent,
```

Comment:
You are declaring EventsComponent here (AppModule) and in the EventsModule. This should only be part of the declaration of EventsModule.

Reply:
done